### PR TITLE
fix: unstable front-end tests

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

**Why submit this pull request?**
- [x] Bugfix

**Related issues**
related #1605 
___
fixed unstable front-end tests.

### CODE REVIEW

File: web/cypress/integration/route/import_export_route.spec.js [hunk 1]
```diff
@@ -124,14 +124,13 @@ context('import and export routes', () => {
...
     cy.visit('/routes/list');
+    cy.get(this.domSelector.refresh).click();
+
     for (let i = 0; i < 2; i += 1) {
-      cy.contains(data[`route_name_${i}`])
-        .siblings()
-        .contains(componentLocaleUS['component.global.delete'])
-        .click();
-      cy.contains('button', componentLocaleUS['component.global.confirm']).click();
+      cy.contains('Delete').should('be.visible').click();
```
Comment (LiteSun) [inline web/cypress/integration/route/import_export_route.spec.js:130]: There will be two delete buttons here, and you should locate the specific delete button IMO.

Comment (guoqqqi) [reply web/cypress/integration/route/import_export_route.spec.js:130]: I don't think there is a need to find a specific delete button. Contains will only find the first element that meets the conditions.
reference: https://docs.cypress.io/api/commands/contains.html#Single-Element

Comment (LiteSun) [reply web/cypress/integration/route/import_export_route.spec.js:130]: I know the rule(only find the first element), but it's too tricky to write this way and not very friendly to new contributors IMO🤔

Comment (juzhiyuan) [reply web/cypress/integration/route/import_export_route.spec.js:130]: Would better to add more information about this Button.

Comment (guoqqqi) [reply web/cypress/integration/route/import_export_route.spec.js]: All right.

**Review by LiteSun — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by bzp2010 — APPROVED** [review]

**Review by Jaycean — APPROVED** [review]
LGTM

**Pull request merged**